sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -2434,6 +2440,7 @@ def __init__(
       gcs_location=None,
       method=None,
       use_native_datetime=False,
```

Comment:
Just noticed this parameter. According to the docs by default DATETIME columns are returned as strings, if this parameter is True they're returned as Python datetime instances. We'll need to account for this in the schema we generate.